src/main/lrsql/admin/routes.clj

```diff
@@ -20,7 +20,8 @@
    (i/lrs-interceptor lrs)])
 
 (defn admin-account-routes
-  [common-interceptors jwt-secret jwt-exp jwt-leeway]
+  [common-interceptors jwt-secret jwt-exp jwt-leeway
+   no-val? no-val-uname no-val-issuer no-val-role-key no-val-role]
```

Comment:
We can probably put all the `no-val*` args into their own opt map arg for `admin-account-routes` and `admin-cred-routes` as well.

Reply:
agree with the opt map idea, anything to kill the clutter here, but can you illustrate what you mean? not familiar with the clojure change.

Reply:
Mockup of the idea with `validate-jwt`:

```clojure
(let [no-val-opts {:no-val? no-val?
                   :no-val-uname no-val-uname
                   :no-val-issuer no-val-issuer
                   :no-val-role-key no-val-role-key
                   :no-val-role no-val-role)]
  (validate-jwt jwt-secret jwt-leeway no-val-opts))
```

Basically you'd pass the opt map as if the function signature is `(validate-jwt jwt-secret jwt-leeway {...})`, without the `&` indicating kwargs.

Reply:
https://github.com/yetanalytics/lrsql/pull/324/commits/563b7a7298f8fda83ec087e46a5f47726589bda8

Reply:
You couldn't freely pass opt maps where kwargs are supposed to be before 1.11, no.